codebase/composer.json

```diff
@@ -69,27 +74,27 @@
         "drupal/simple_content_notifications": "^1.0",
         "drupal/smtp": "^1.2",
         "drupal/structure_sync": "^2.0",
-        "drupal/tac_lite": "^1.6",
+        "drupal/tac_lite": "^1.7",
         "drupal/taxonomy_manager": "^2.0",
         "drupal/term_merge": "^2.0@beta",
-        "drupal/tfa": "^1.0",
+        "drupal/tfa": "^1.3",
         "drupal/transliterate_filenames": "^2.0",
         "drupal/twig_tweak": "^3.2",
         "drupal/user_redirect": "^2.0",
+        "drupal/variationcache": "^1.3",
         "drupal/view_unpublished": "^1.1",
         "drupal/views_data_export": "^1.2",
         "drupal/views_field_view": "^1.0@beta",
         "drupal/views_node_access_filter": "^1.0@beta",
         "drupal/workbench_email": "^3.0",
-        "drush/drush": "^10.3",
+        "drush/drush": "^12",
         "islandora-rdm/islandora_fits": "dev-8.x-1.x as 1.x-dev",
-        "islandora/advanced_search": "^2.0.0",
         "islandora/controlled_access_terms": "^2",
-        "islandora/islandora": "^2",
-        "islandora/openseadragon": "^2",
+        "islandora/islandora": "^2.8.1",
+        "islandora/islandora_mirador": "^2",
         "islandora/views_nested_details": "^1.0",
         "library/pdf.js": "^2.4",
-        "mjordan/islandora_workbench_integration": "^1.0"
+        "mjordan/islandora_workbench_integration": "dev-main"
```

Comment:
Is this the right version?

Reply:
assume so as it's the version in the latest starter template, https://github.com/Islandora-Devops/islandora-starter-site/blob/main/composer.json